### PR TITLE
Add dark and lightlevel properties to motion sensors light sensor

### PR DESCRIPTION
**Pull request by tim-hellhake**
Resolves https://github.com/mozilla-iot/philips-hue-adapter/issues/43
image

### CODE REVIEW

**Comment by tim-hellhake** [comment]
image

File: philips-hue-adapter.js [hunk 1]
```diff
@@ -216,20 +216,46 @@ class PhilipsHueDevice extends Device {
...
+            device.state.dark));
+
+        this.properties.set(
+          'lightlevel',
+          new PhilipsHueProperty(
+            this,
+            'lightlevel',
+            {
+              '@type': 'LevelProperty',
+              label: 'Lightlevel',
```
Comment (hobinjk) [inline philips-hue-adapter.js]: nitpick: "Light Level" reads better as a label

Comment (tim-hellhake) [reply philips-hue-adapter.js]: Agreed

Comment (tim-hellhake) [reply philips-hue-adapter.js]: Done

**Comment by tim-hellhake** [comment]
@hobinjk Anything left to change here?

**Comment by hobinjk** [comment]
Sorry, this got lost in my inbox. Tested locally and everything LGTM!

**Pull request merged**